importScripts/refineryParameters/UsualLocationLowest.ca_storage_locations.usual.storageLocationSplitter.json

```diff
@@ -0,0 +1,47 @@
+{
+    "storageLocationType": "^UsualLocationType",
```

Comment:
previously `UsualLocation` was a single value, not hierarchical columns. We need a more complicated `storageLocationSplitter` to split out the values.